### PR TITLE
CKO-(1451, 1635) Widget and segment integration in iOS SDK

### PR DESCRIPTION
**Pull request by alex-lane**

### CODE REVIEW

File: QuadPaySDK.podspec [hunk 1]
```diff
@@ -1,13 +1,17 @@
 Pod::Spec.new do |s|
   s.name						= 'QuadPaySDK'
-  s.authors						= "QuadPay, Inc."
-  s.version						= '0.4.0'
-  s.summary						= 'Integrate QuadPay into your iOS app'
+  s.authors						= "Zip Co."
+  s.version						= '0.5.10-beta'
```
Comment (alex-lane) [inline QuadPaySDK.podspec:4]: This shouldn't be beta for the master PR. Maybe just `0.5`?

File: QuadPaySDK.podspec [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+  s.summary						= 'Integrate Zip (formally QuadPay) into your iOS app'
   s.homepage					= 'https://github.com/quadpay/quadpay-merchant-sdk-ios'
   s.license						= { :type => "mit", :file => "LICENSE" }
-  s.source						= { :git => 'https://github.com/quadpay/quadpay-merchant-sdk-ios.git', :tag => s.version.to_s }
-  s.ios.deployment_target		= '9.0'
-  s.source_files				    = 'QuadPaySDK/**/*.{h,m}'
+  s.source						= { :git => 'https://github.com/quadpay/quadpay-merchant-sdk-ios.git', :tag => s.version.to_s, :branch => 'widget-and-segment' }
```
Comment (alex-lane) [inline QuadPaySDK.podspec:8]: Need to remove the branch name.

File: QuadPaySDK/QuadPaySDKKeys.m [hunk 3]
```diff
@@ -0,0 +1,62 @@
+//
+// Generated by CocoaPods-Keys
+// on 17/10/2022
+// For more information see https://github.com/orta/cocoapods-keys
+//
+
+#import <Foundation/NSDictionary.h>
+#import "QuadPaySDKKeys.h"
+
+@interface QuadPaySDKKeys ()
```
Comment (alex-lane) [inline QuadPaySDK/QuadPaySDKKeys.m:10]: These keys are set to the development values. They need to be recreated to the production values.

**Comment by alex-lane** [comment]
I updated the keys and version number. Should be good for review now.

File: QuadPaySDK/www/index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                size=''
+                widgetVerbiage=''
+                subtextLayout=''
+                hideLogo=''
+                implementCloseModal=''
+                currency=''
+                languageSelector=''>
+        </quadpay-widget-v3>
+    </body>
+
```
[the remark this answers is not in the document]
Comment (PetrosZip) [reply QuadPaySDK/www/index.html:36]: I am adding this change on Android as i am doing a testing now, i can do the change here as well

[the remark this answers is not in the document]
Comment (alex-lane) [reply QuadPaySDK/www/index.html:36]: Ok, thanks.

**Review by rhyswilliamszip — APPROVED** [review]
One comment

**Review by rhyswilliamszip — APPROVED** [review]

**Review by rhyswilliamszip — APPROVED** [review]

**Pull request merged**